components/card.css

```diff
@@ -0,0 +1,130 @@
+@import url('../styles.css');
+
+.card{
+   width: 17rem;
+   box-shadow: 3px 1px 6px 1px rgb(0 0 0 / 20%);
+   border-radius: 10px;
+   margin-left: 4rem;
+   transition: .2s all ease-in-out;
+}
+
+.card:hover{
+    box-shadow: 3px 1px 10px 1px rgb(0 0 0 / 20%);;
+}
+
+.img-content{
+    margin-top: -1rem;
+}
+
+.card img{
+    height: 100%;
+    width: 100%;
+}
+
+.card-content{
+    padding: 0.5rem;
+}
+
+.text-badge{
+    position: relative;
+    top: 2rem;
+    background-color: #E67A00;
```

Comment:
You can use CSS variable here for background-color

Reply:
thanks, Nikhil adding the variable for bg color.